tests/API/Integration/DepartmentsTests.cs

```diff
@@ -99,6 +99,49 @@ public async Task GetAuditorParksMemberUnits()
 				Assert.That(actual.First().Id, Is.EqualTo(expected.First().Id));
 				Assert.That(actual.First().Parent.Id, Is.EqualTo(expected.First().Parent.Id));
 			}
+
+			[Test]
+			public async Task GetsDistinctMemberUnits()
+			{
+				// Add Chris as a second member of the Auditor team.
+				var db = Database.PeopleContext.Create(Database.PeopleContext.LocalDatabaseConnectionString);
+				var chris = new Person
+				{
+					Netid = "ctraeger",
+					Name = "Traeger, Chris",
+					NameFirst = "Chris",
+					NameLast = "Traeger",
+					Position = "Sr. Auditor",
+					Location = "Pawnee",
+					Campus = "Pawnee",
+					CampusPhone = "5-6789",
+					CampusEmail = "ctraeger@auditz.com",
+					Notes = "",
+					PhotoUrl = "",
+					Responsibilities = Responsibilities.None,
+					DepartmentId = TestEntities.Departments.AuditorId,
```

Comment:
I think its `DepartmentId = TestEntities.Departments.Auditor.Id`?

Reply:
They should be the same thing.  `TestEntities.Departments.Auditor.Id` is set using `TestEntities.Departments.AuditorId`.
Feel free to change it if you think it's easier to read the other way.

Reply:
Thanks, Jason.